frontend/src/component/filter/Filters/Filters.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { FILTER_ITEM } from 'utils/testIds';
+import { Filters, IFilterItem } from './Filters';
+
+test('shoulder render all available filters', async () => {
+    const availableFilters: IFilterItem[] = [
```

Comment:
can we extract the fields that are not relevant into meaningful variables and only explicitly name the ones that affect the test?
e.g. const irrelevantOperators = ['irrelevant1', 'irrelevant2']
singularOperators: irrelevantsOperators

Reply:
Done

Reply:
same in the other test

Reply:
Damn, it seems violent, but I guess its for the greater good.